ta/pkcs11/src/object.c

```diff
@@ -47,6 +47,23 @@ static struct ck_token *get_session_token(void *session)
 	return pkcs11_session2token(ck_session);
 }
 
+bool check_object_access_against_session(struct pkcs11_object *object,
```

Comment:
This function will always be called after `pkcs11_handle2object()` find an object => should be merged into `pkcs11_handle2object()` straight.

Reply:
Let's see how that works.

Reply:
I believe it cleaned it up.